src/zkevm_specs/util/gas.py

```diff
@@ -0,0 +1,43 @@
+from ..encoding import U8, U64, U128, U256
+from typing import Sequence, Tuple
+
+G_MEM = 3
```

Comment:
Doc constants please.

Reply:
this constant is already define at [here](https://github.com/appliedzkp/zkevm-specs/blob/master/src%2Fzkevm_specs%2Futil%2Fparam.py#L70)

Reply:
updated in [0912af8](https://github.com/appliedzkp/zkevm-specs/pull/125/commits/0912af8352fad807c9c70e00c0cd89c1e7532ded)